flow-nodes/api-builder-plugin-logger/package.json

```diff
@@ -20,15 +20,16 @@
 		"url": "https://github.com/axway/api-builder-examples"
 	},
 	"dependencies": {
-		"@axway/api-builder-sdk": "^0.2.0"
+		"@axway/api-builder-sdk": "^0.8.0"
 	},
 	"peerDependencies": {
 		"@axway/api-builder-runtime": "^4.5.0"
 	},
 	"devDependencies": {
 		"chai": "^4.1.2",
 		"mocha": "^6.1.4",
-		"simple-mock": "^0.8.0"
+		"simple-mock": "^0.8.0",
+		"@axway/api-builder-test-utils": "0.1.0"
```

Comment:
Same comment as on rdpp-6419 about alphabetical.  Also, ^0.1.0.

Reply:
Sorted it in alphabetical order.